### PR TITLE
merge giant adenofibroma of the breast

### LINKED ISSUE #7047: giant adenofibroma [of the] breast MONDO:0015872 & MONDO:0004150[Merge]
**Mondo term (ID and Label)**
MONDO:0015872 giant adenofibroma of the breast

**Reason for deprecation**
Strings are almost same, so to me as a human they seem to be the same concept. 
UMLS maps Orpha [180267](http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=180267) to same CUI as the SNOMED CT (http://identifiers.org/snomedct/254846003)
UMLS CUI is C0346157

**Term to be merged with**
MONDO:0004150 breast giant fibroadenoma

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other notes:**
[C0334500](http://linkedlifedata.com/resource/umls/id/C0334500) =="giant fibroadenoma"
 Is not exact match to "giant adenofibroma" _located in breast_, if it's not updated/removed from the UMLS update procedure underway in another ticket, it should be removed from this Mondo ID.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7047

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -72135,17 +72135,30 @@ is_a: MONDO:0006215 {source="DOID:7222"} ! gallbladder adenocarcinoma
 id: MONDO:0004150
 name: breast giant fibroadenoma
 def: "A breast fibroadenoma characterized by a very large size. This term has also been used as a synonym for juvenile fibroadenoma by some authors. The latter is characterized by epithelial hyperplasia and an increased stromal cellularity." [NCIT:C4273]
+subset: gard_rare {source="GARD:20206"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:180267"}
+subset: orphanet_rare {source="Orphanet:180267"}
+subset: rare
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:72142]: Should all these "subset" tags be added (similar to question on another PR)?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -72135,17 +72135,30 @@ is_a: MONDO:0006215 {source="DOID:7222"} ! gallbladder adenocarcinoma
...
 synonym: "giant fibroadenoma of the breast" EXACT [NCIT:C4273]
 xref: DOID:7223 {source="MONDO:equivalentTo"}
+xref: GARD:20206 {source="Orphanet:180267"}
+xref: ICD10CM:D24 {source="Orphanet:180267/ntbt", source="Orphanet:180267"}
 xref: ICDO:9016/0 {source="NCIT:C4273"}
 xref: NCIT:C4273 {source="MONDO:equivalentTo", source="MONDO:exact-label-match", source="DOID:7223"}
+xref: Orphanet:180267 {source="MONDO:equivalentTo"}
 xref: SCTID:254846003 {source="MONDO:equivalentTo", source="DOID:7223"}
 xref: UMLS:C0346157 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="DOID:7223"}
+xref: UMLS:CN200476
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:72157]: does this need a source?

**Review by twhetzel — CHANGES_REQUESTED** [review]
two comments to clarify

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:72142]: @sabrinatoro should subset tags be moved over from a merged term or deleted?

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**